submissions/cynthiaNwakaeme/CSS/style.css

```diff
@@ -37,7 +37,8 @@ header #nav-branding{
     float: left;
```

Comment:
Which html file is using this file? @Cynthia-Berry

Reply:
No need to remove it.

Reply:
Thank you so much.
i appreciate.